cmd/server/http/http.go

```diff
@@ -728,3 +726,26 @@ func release(payload *payload, flowSvc *flow.Service) http.HandlerFunc {
 func convertTimeToEpoch(t time.Time) int64 {
 	return t.UnixNano() / int64(time.Millisecond)
 }
+
+type commitInfo struct {
+	AuthorName  string
+	AuthorEmail string
+	Service     string
+}
+
+func extractAuthorFromCommit(message string) (commitInfo, error) {
```

Comment:
You use two names for this function. `extractAuthorFromCommit` and in the test its `commitMessageExtraction`.
I think the latter is more precise, but maybe something like `extractInfoFromCommit` returning `commitInfo` reads well?

Reply:
Yes it does. Fixed 👍